added README-de-DE.md and updated README.md (#522)
thamara/time-to-leave: docs/README-de-DE.md

<div align="center">
  <img src="../assets/timetoleave.png" alt="Time to Leave Logo">

>   <p>Notieren Sie Ihre Arbeitszeiten und werden Sie benachrichtigt, wenn es Zeit ist, das Büro zu verlassen und zu leben</p>

Reviewer: ```suggestion
  <p>Logge deine Arbeitszeiten und werde benachrichtigt, wenn es Zeit ist, das Büro zu verlassen und dein Leben zu genießen</p>
```

  <br/>

<img src="https://img.shields.io/badge/platforms-Windows%20%7C%20MacOS%20%7C%20Linux-green" alt="Platform">

> <img src="https://img.shields.io/github/downloads/thamara/time-to-leave/total" alt="Downloads in Total">

Reviewer: ```suggestion
<img src="https://img.shields.io/github/downloads/thamara/time-to-leave/total" alt="Downloads gesamt">
```

> <a href="https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5"><img src="https://img.shields.io/github/v/release/thamara/time-to-leave" alt="Latest Release"></a>

Reviewer: ```suggestion
<a href="https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5"><img src="https://img.shields.io/github/v/release/thamara/time-to-leave" alt="Neueste Version"></a>
```

<img src="https://img.shields.io/github/workflow/status/thamara/time-to-leave/Code%20Coverage" alt="Build">

> <a href="http://makeapullrequest.com/"><img src="https://img.shields.io/badge/PRs-welcome-purple" alt="Latest Release"></a>

Reviewer: ```suggestion
<a href="http://makeapullrequest.com/"><img src="https://img.shields.io/badge/PRs-welcome-purple" alt="Neueste Version"></a>
```

   <br/>

  <img src="https://user-images.githubusercontent.com/3754225/94519528-4e549900-0248-11eb-8872-b6fb2d47f43c.jpg" alt="Time to Leave Screenshot">

  <br/>

</div>

---

> Logge die Zeit, zu der du heute mit der Arbeit begonnen hast und das Programm berechnet unter Berücksichtigung der Zeit, die du beim Mitagessen verbracht hast, wann du das Büro verlassen solltest. Wenn es Zeit ist zu gehen, werden Sie vom Programm benachrichtigt.

Reviewer: ```suggestion
Logge die Zeit, zu der du heute mit der Arbeit begonnen hast und das Programm berechnet unter Berücksichtigung der Zeit, die du beim Mitagessen verbracht hast, wann du das Büro verlassen solltest. Wenn es Zeit ist zu gehen, wirst du vom Programm benachrichtigt.
```

> Notieren Sie sich die Zeit, zu der Sie heute mit der Arbeit begonnen haben, und das Programm berechnet die Zeit, die Sie das Büro verlassen sollten, unter Berücksichtigung der Zeit, die Sie beim Mittagessen verbracht haben. Wenn es Zeit ist, werden Sie vom Programm benachrichtigt.

Reviewer: ```suggestion
Logge die Zeit, zu der du heute mit der Arbeit begonnen hast und das Programm berechnet unter Berücksichtigung der Zeit, die du beim Mitagessen verbracht hast, wann du das Büro verlassen solltest. Wenn es Zeit ist zu gehen, werden Sie vom Programm benachrichtigt.
```

![time-to-leave-notification](https://user-images.githubusercontent.com/3754225/94519526-4dbc0280-0248-11eb-9738-ffae936cfa4a.jpg)

> Sie können Ihre Einstellungen steuern, indem Sie Ihre Arbeitstage eingeben, wie viele Stunden Sie pro Tag arbeiten und ob Sie benachrichtigt werden möchten oder nicht.

Reviewer: ```suggestion
Du kannst Einstellungen setzen indem du deine Arbeitstage angibst, wieviele Stunden pro Tag du arbeitest und ob du benachrichtigt werden möchtest oder nicht.
```

![time-to-leave-settings](https://user-images.githubusercontent.com/3754225/94519531-4eed2f80-0248-11eb-9303-78f9abe69201.jpg)

> Sie können auch Freizeit an den Tagen hinzufügen, an denen Sie nicht gearbeitet haben.

Reviewer: ```suggestion
Du kannst Tage an denen du nicht gearbeitet hast als Freizeit festlegen.
```

![time-to-leave-waiver](https://user-images.githubusercontent.com/3754225/94762058-4e79a380-03c4-11eb-8f28-1c480dbf8b5c.png)

## So installieren Sie TTL

> Time to Leave funktioniert unter MacOS, Windows und Linux, und Sie können die gewünschte Version von der herunterladen. [neuesten Version](https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5).

Reviewer: ```suggestion
Time to Leave ist kompatibel mit MacOS, Windows und Linux. Du kannst die gewünschte Version vom [neuesten Release an herunterladen](https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5).
```

## Wie man TTL baut und dazu beiträgt

> Wenn Sie helfen möchten, lesen Sie unseren Leitfaden [Beitrag](../CONTRIBUTING.md).

Reviewer: ```suggestion
Wenn du helfen möchtest, lese unseren [Contribution-Leitfaden](../CONTRIBUTING.md).
```

> Sie können auch den verwenden [Entwicklungsleitfaden](../DEVELOPMENT.md) für mehr Informationen.

Reviewer: ```suggestion
Du kannst auch den [Entwicklungsleitfaden](../DEVELOPMENT.md) für mehr Informationen verwenden.
```

> ## In Kontakt kommen!

Reviewer: ```suggestion
## In Kontakt treten!
```

> Tritt unser ... bei [Kanal der Zwietracht](https://discord.gg/P3KkEF5) um neue Funktionen und mehr zu diskutieren.

Reviewer: ```suggestion
Tritt unserem [Discord Server](https://discord.gg/P3KkEF5) bei, um über neue Funktion und mehr zu diskutieren.
```

> ## Créditos

Reviewer: ```suggestion
## Credits
```

> Von [freepik](https://www.flaticon.com/authors/freepik) von [www.flaticoncom](https://www.flaticon.com) erstellte Symbole.

Reviewer: ```suggestion
Icons von [freepik](https://www.flaticon.com/authors/freepik) von [www.flaticoncom](https://www.flaticon.com).
```

> Uhrensymbol von [lucy-g](https://icon-icons.com/icon/timer/121243), Farbvariationen und daraus modifizierte Formate.

Reviewer: ```suggestion
Originales Timer Icon sowie Variationen und modifizierte Formate von [lucy-g](https://icon-icons.com/icon/timer/121243).
```

> Patientensymbol von [SVG Repo](https://www.svgrepo.com/svg/271898/sick).

Reviewer: ```suggestion
Kranken-Icon von [SVG Repo](https://www.svgrepo.com/svg/271898/sick).
```

> Symbol für den Displayaustausch von [Pixel perfect](https://www.flaticon.com/authors/pixel-perfect) von [www.flaticon.com](https://www.flaticon.com).

Reviewer: ```suggestion
"View Switch" Icon von [Pixel perfect](https://www.flaticon.com/authors/pixel-perfect) von [www.flaticon.com](https://www.flaticon.com).
```
